operator/apis/platform/v1alpha1/central_types.go

```diff
@@ -109,6 +109,12 @@ type CentralComponentSpec struct {
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=4
 	Persistence *Persistence `json:"persistence,omitempty"`
 
+	// Enable necessary components for provisioning of monitoring metrics for
+	// Central and other components of ACS stack. This is required if metrics
+	// are collected by Observability Operator.
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=5
+	ExposeMonitoring *bool `json:"exposeMonitoring,omitempty"`
```

Comment:
Please [avoid booleans](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#primitive-types), instead use a "typed string" idiom, see `ScannerComponentPolicy` below for an example.

It might also make sense to consider introducing a `Monitoring` sub-struct and place the knob there, in case we plan to introduce more related knobs in the future (you'd probably know better if there would be anything else).

Another thing is exposing monitoring for `scanner`. AFAICT there is functionality for a metrics endpoint in scanner at the moment, but if we also plan to add it in the future, then perhaps it makes more sense to add this `Monitoring` sub-struct at the `spec` level?